src/open_petro_elastic/material/span_wagner/carbon_dioxide.py

```diff
@@ -0,0 +1,551 @@
+"""
+Equation 6.1 from Span & Wagner [2], and its derivatives have been computed using sympy and the following code:
+
+First, define the kernels of the summations in the Helmholtz energy residual function of Table 32:
+    >>> import sympy as sp
+    ... (n1, t1, d1, n2, d2, t2, c2, n3, d3, t3, alpha3, epsilon3, beta3, gamma3, n4, b4, a4, beta4, A4, B4, C4, D4) =\
+    ...     sp.symbols('n1 t1 d1 n2 d2 t2 c2 n3 d3 t3 alpha3 epsilon3 beta3 gamma3 n4 b4 a4 beta4 A4 B4 C4 D4')
+    ... tau, delta = sp.symbols('tau delta', real=True)
+    ... i = sp.symbols('i', integer=True)
+    ... s1 = n1 * delta ** d1 * tau ** t1
```

Comment:
It seems that if evaluating these expressions over `np.ndarray`s that could be useful for having a bit cleaner code in this module. Regardless, probably moving the sympy part to a separate file, for instance `src/open_petro_elastic/material/span_wagner/equations.py`, seems like a good idea.

Reply:
Implementation with direct use of sympy for evaluation is now in place.